### ISSUE TITLE
Does this make my notebook slow/lag/sluggish?

### ISSUE THREAD

**Issue by guojunchi11** [jcb91/jupyter_highlight_selected_word#19]
As the notebook accumulates more content after working on it for a while, does this feature make my notebook slow/lag/sluggish?  Such as longer lagging time when typing in codes.

**Comment by jcb91** [comment]
It's certainly possible, since the more cells there are in the notebook, the more time will be spent updating the highlighting on each of them. How much content are we talking here?

**Comment by jcb91** [comment]
I guess it might be possible to limit highlighting efforts to cells currently scrolled into view, but then you'd need to update every time someone scrolled as well...

**Comment by guojunchi11** [comment]
Yeah, I was suffering lagging time when typing,  it does not take much content to slow down...
I disabled it, and the norm speed resumes,
limiting within in one cell seems to be a reasonable option to try.
